match/lib/match/options.rb

```diff
@@ -55,6 +55,12 @@ def self.available_options
                                      description: "Only fetch existing certificates and profiles, don't generate new ones",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :disable_encryption,
+                                     env_name: "MATCH_DISABLE_ENCRYPTION",
+                                     short_option: "-d",
+                                     description: "Disable encryption, certificates including keys and profiles won't be encrypted! be sure that your git repo is safe!",
```

Comment:
`Be` should be upper case. Could you extend the description to mention here that this can't be changed once _match_ is set up?